test/iron-location.html

```diff
@@ -75,6 +88,18 @@
 <script>
   'use strict';
 
+  // IE baseURI workaround
+  function getBaseURI() {
```

Comment:
you're better off polyfilling this rather than having a method, e.g. like suggested here https://github.com/webcomponents/webcomponents-platform/issues/2